lib/Crypto/doc/sailfish-crypto-plugins.qdoc

```diff
@@ -0,0 +1,92 @@
+/****************************************************************************************
+**
+** Copyright (C) 2018 Jolla Ltd.
+** Contact: Chris Adams <chris.adams@jollamobile.com>
+** All rights reserved.
+**
+****************************************************************************************/
+
+/*!
+\contentspage {Sailfish OS Crypto Plugins Contents}
+\page sailfish-crypto-plugins.html
+
+\title Default Crypto Plugins for the Sailfish OS Secrets and Crypto Framework
+
+A number of plugins have been written for the framework which provide
+a variety of functionality for clients to use.  Most clients should use
+the platform default plugins when writing their applications, as these
+will provide the most consistent and secure experience.
+
+Device vendors and trusted partners may wish to provide their own plugins,
+and some applications may wish to specifically use those plugins.
+
+\section1 Default Crypto Plugins
+
+Currently, there are two Sailfish OS Crypto plugins shipped by default:
+
+\list
+\li org.sailfishos.crypto.plugin.crypto.openssl
+\li org.sailfishos.secrets.plugin.encryptedstorage.sqlcipher
+\endlist
+
+The first plugin provides implementations of common cryptography functionality
+by calling the OpenSSL library functions internally.
+
+The second plugin shares its cryptography implementation with the first plugin,
+but also provides block-level encrypted secure storage of collections of
+secrets, using the SQLCipher library for its database operations.  Thus it is
+both a \tt{Sailfish::Crypto::CryptoPlugin} and a
+\tt{Sailfish::Secrets::EncryptedStoragePlugin}.
+
+Note that neither of these plugins use TEE/TPM or other secure-hardware
```

Comment:
Maybe TEE/TPM -> TEE / TPM. I wonder if we should write TEE/TPM in open form, not every knows what they mean. Though the subsequent phrase "or other secure-hardware" does give some context.

Reply:
I think it's fine, this is a secure storage and cryptography API...